base_tier_validation_server_action/models/tier_review.py

```diff
@@ -7,24 +7,33 @@
 class TierReview(models.Model):
     _inherit = "tier.review"
 
+    def _run_server_action(self):
+        for rec in self.filtered(
```

Comment:
Since it is quite complicated. Can we add some comment here and there? WDYT?

Reply:
no comment? :)